nuget/nunitlite.nuspec

```diff
@@ -30,8 +30,8 @@
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
     <file src="bin/sl-5.0/nunitlite.dll" target="lib/sl5" />
-    <file src="../../nuget/Program.cs" target="content" />
-    <file src="../../nuget/Program.vb" target="content" />
-    <file src="../../nuget/install.ps1" target="tools" />
+    <file src="..\..\nuget\Program.cs" target="content" />
```

Comment:
This fix for Linux didn't work on Windows. The three files were not included in the NuGet package and even worse, Program.cs was removed when I upgraded. This fixes it, but we will need to revisit #760 if packaging is important on Linux.

It looks like relative paths don't work with non-Windows separators, so we will likely need to copy these three files into the image directory.